proto/pdpb.proto

```diff
@@ -354,6 +354,7 @@ message StoreHeartbeatRequest {
     RequestHeader header = 1;
 
     StoreStats stats = 2;
+    uint64 timestamp = 3;
```

Comment:
`interval` or `duration`? Also please add some comments.

Reply:
There is a well-known `Timestamp`[1] message.

[1] https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Timestamp